src/test/groovy/org/prebid/server/functional/util/privacy/gpp/UspVaV1Consent.groovy

```diff
@@ -16,6 +19,49 @@ class UspVaV1Consent extends GppConsent {
         gppModel.encodeSection(SECTION.name)
     }
 
+    @Override
+    UspNatV1Consent normaliseToNational() {
+        def uspVaV1 = ((UspVaV1) this.gppModel.getSection(SECTION.name))
+
+        new UspNatV1Consent.Builder()
+                .setSensitiveDataProcessing(normaliseSensitiveData(uspVaV1))
+                .setKnownChildSensitiveDataConsents(normalizeChildConsents(uspVaV1))
+                .setSaleOptOutNotice(uspVaV1.saleOptOutNotice)
+                .setTargetedAdvertisingOptOutNotice(uspVaV1.targetedAdvertisingOptOutNotice)
+                .setTargetedAdvertisingOptOut(uspVaV1.targetedAdvertisingOptOut)
+                .setSharingNotice(uspVaV1.sharingNotice)
+                .setSaleOptOut(uspVaV1.saleOptOut)
+                .setMspaCoveredTransaction(uspVaV1.mspaCoveredTransaction)
+                .setMspaOptOutOptionMode(uspVaV1.mspaOptOutOptionMode)
+                .setMspaServiceProviderMode(uspVaV1.mspaServiceProviderMode)
+                .build() as UspNatV1Consent
+    }
+
+    @Override
+    protected UsNationalSensitiveData normaliseSensitiveData(EncodableSection uspVaV1) {
+        def virginiaSensitiveData = UsVirginiaSensitiveData.fromList(((UspVaV1)uspVaV1).sensitiveDataProcessing)
+
+        new UsNationalSensitiveData().tap {
+            racialEthnicOrigin = virginiaSensitiveData.racialEthnicOrigin
+            religiousBeliefs = virginiaSensitiveData.religiousBeliefs
+            healthInfo = virginiaSensitiveData.healthInfo
+            orientation = virginiaSensitiveData.orientation
+            citizenshipStatus = virginiaSensitiveData.citizenshipStatus
+            geneticId = virginiaSensitiveData.geneticId
+            biometricId = virginiaSensitiveData.biometricId
+            geolocation = virginiaSensitiveData.geolocation
+            idNumbers = 0
```

Comment:
Magic number

Reply:
Updated, but also be aware that I returned:
```
    SENSITIVE_DATA_RACIAL_ETHNIC_ORIGIN(UspNatV1Field.SENSITIVE_DATA_PROCESSING + 1),
    SENSITIVE_DATA_RELIGIOUS_BELIEFS(UspNatV1Field.SENSITIVE_DATA_PROCESSING + 2),
```
It's required for normalizing tests